sycl/test-e2e/Matrix/get_coord_int8_matB.cpp

```diff
@@ -9,7 +9,6 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: *
```

Comment:
@Nuullll, can you please comment on whether we should keep xfail::cpu here?
"get_coord" was also corrected on the CPU for packed B, right?

Reply:
"XFAIL: cpu" should be kept here.